sass/main.scss

```diff
@@ -0,0 +1,177 @@
+*{
+  margin: 0px;
+  padding: 0px;
+}
+
+#topbar{
+  width: 100%;
+  height: 50px;
+  // margin: auto;
+  overflow: hidden;
+  padding: auto;
+  background-color: #0d47a1;
+}
+
+// #branding{
+//   margin-top: 10px;
+//   margin-bottom: 10px;
+// }
+
+#topbarh1{
```

Comment:
This element id name is sort of fuzzy. You should make your variables clear and unambiguous.